lib/cisco_node_utils/vxlan_vtep_vni.rb

```diff
@@ -24,13 +24,19 @@
 module Cisco
   # VxlanVtepVni - node utility for vxlan vtep vni members.
   class VxlanVtepVni < NodeUtil
-    attr_reader :name, :vni
+    attr_reader :name, :vni, :vrf
 
-    def initialize(name, vni, instantiate=true)
+    def initialize(name, vni, vrf=false, instantiate=true)
```

Comment:
Yeah.. glad you made the comment.  I went aback and forth with several names before I settled on `vrf` but based on your suggestion I agree that `assoc_vrf` adds more clarity.